manifest.json

```diff
@@ -1,20 +1,24 @@
 {
   "name": "KillProof Module",
-  "version": "1.5.1",
-  "namespace": "Nekres.KillProof",
+  "version": "1.5.2",
+  "namespace": "KillProofModule",
```

Comment:
Careful with changing the package namespace, FYI.

https://github.com/blish-hud/manifest.json/blob/master/manifest-v1.md#namespace

It resets any internal settings saved for your module for the user, if it's enabled, etc.

Reply:
Are settings being used right now anyways?  If not, biggest thing it'll even do is just disable the module for anyone that already had it enabled.

Reply:
Just if QuickMenu is enabled.